opcua/common/instantiate.py

```diff
@@ -84,14 +77,18 @@ def _instantiate_node(server, parentid, rdesc, nodeid, bname, dname=None, recurs
             for c_rdesc in descs:
                 # skip items that already exists, prefer the 'lowest' one in object hierarchy
                 if not ua_utils.is_child_present(node, c_rdesc.BrowseName):
+                    c_node_type = Node(server, c_rdesc.NodeId)
+                    # skip optional elements, or node without ModellingRule at top-level
```

Comment:
should we skip or raise an exception?

Reply:
The standard wants them to be excluded. There are use cases that needs this behavior, e.g. in Part 10 (Programs).
So nodes without modeling rule are static members of the type. For nodes with optional modellling the server might include or skip them.

Reply:
ok

Reply:
maybe log with logger.info...